servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/CatalogTests.java

```diff
@@ -1972,15 +1971,20 @@ public void testCompleteCreateOrReplaceTransactionReplace() {
 
     Map<String, String> properties =
         ImmutableMap.of("user", "someone", "created-at", "2022-02-25T00:38:19");
-    String location = temporaryLocation();
-    Transaction createOrReplace =
+
+    TableBuilder tableBuilder =
         catalog
             .buildTable(TABLE, SCHEMA)
-            .withLocation(location)
             .withPartitionSpec(SPEC)
             .withSortOrder(WRITE_ORDER)
-            .withProperties(properties)
-            .createOrReplaceTransaction();
+            .withProperties(properties);
+
+    String location = temporaryLocation();
```

Comment:
I changed 2 or 3 tests like this – I think the change makes sense: if the catalog overrides locations, the client should not call `withLocation` nor write files to a random location before the catalog gets a chance to properly create the table.

Reply:
Yup - that's the change I had in mind.
However, the call to `temporaryLocation()` can be moved inside the `if`-body.